apps/store/themes/store/js/categorization.js

```diff
@@ -17,39 +17,62 @@
  */
 
 var categorizationArray = [];
-var categorization = function() {
+var categorization = function () {
 
-    var updateFilters = function (url){
-        var queryArray = url.split("q=")[1].split(",");
+    /**
+     * This method returns the query parameter by name
+     * @param name          name of the query parameter
+     * @param url           url
+     * @returns {string}    query parameter string
+     */
+    var getParameterByName = function (name, url) {
+        if (!url) {
+            url = window.location.href;
+        }
+        var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)");
+        var results = regex.exec(url);
+        if (results && results[2]) {
+            return decodeURIComponent(results[2].replace(/\+/g, " "));
+        }
+
```

Comment:
@sinthuja Thanks for pointing out. This line has been already removed by 0265b89d3db44295f85544d8d209d5cdbc39d56d. Therefore the extra new line does not exist in the latest version of this file.